signers/signer-solana/src/config.ts

```diff
@@ -0,0 +1,13 @@
+type Config = {
+  customRPC: string;
+};
+
+let config: Config | undefined;
+
+export function setConfig(c: { customRPC: string }) {
```

Comment:
Please consider to always type function's return. (helps performance)

Reply:
Tnx, Added.